sections/futures/FeeInfoBox/TradePanelFeeInfo.tsx

```diff
@@ -25,7 +25,7 @@ import TradeTotalFeesRow from './TradeTotalFeesRow';
 
 export const TradePanelFeeInfo = memo(() => {
 	return (
-		<FeeInfoBoxContainer>
+		<FeeInfoBoxContainer $hasBorder={false}>
```

Comment:
Since we separated out the row components I've just been avoiding wrapping them in the InfoBox to get this effect, not sure if you think that's a better solution as it's sort of just voiding the container anyway with this style.

Reply:
True. I'll update this